<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dropbox Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../DropboxIcons.qrc">
    <normaloff>:/icons/img/kfilebox.png</normaloff>:/icons/img/kfilebox.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab0">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="tab0Title">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Desktop integration</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="tab0Spacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="showNotifications">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Show system tray notifications</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="startDaemon">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Start Dropbox on system startup</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="useP2P">
          <property name="text">
           <string>Enable LAN sync</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hideGtkUI">
          <property name="text">
           <string>hide Gtk UI from tray</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Account</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="tab1Title">
          <property name="text">
           <string>Account information for this computer</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="tab1Spacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="titleVersion">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Version</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="displayVersion">
          <property name="text">
           <string>Dropbox v1.0.20</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="titleAccount">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Account</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="displayAccount">
          <property name="text">
           <string>email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Dropbox folder location</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="dropboxFolder">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="moveDropboxFolder">
            <property name="text">
             <string>Move...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QGroupBox" name="downloadBox">
          <property name="title">
           <string>Download</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QRadioButton" name="downloadDontLimitRate">
             <property name="text">
              <string>Don't limit download rate</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="networkDownloadLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="downloadLimitRate">
               <property name="text">
                <string>Limit download rate to:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSpinBox" name="downloadLimitValue">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="downloadLimitValueFor">
               <property name="text">
                <string>kB/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="uploadBox">
          <property name="title">
           <string>Upload</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QRadioButton" name="uploadDontLimitRate">
             <property name="text">
              <string>Don't limit upload rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="uploadAutoLimitRate">
             <property name="text">
              <string>Limit upload rate automatically</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="uploadLimitRate">
               <property name="text">
                <string>Limit upload rate to:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QSpinBox" name="uploadLimitValue">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="value">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="uploadLimitValueFor">
               <property name="text">
                <string>kB/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>Proxy</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="proxyDontUse">
          <property name="text">
           <string>Without proxy</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="proxyAutoDetect">
          <property name="text">
           <string>Auto detected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="proxySetManually">
          <property name="text">
           <string>Manually</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="proxyType">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>HTTP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Socks4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Socks5</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="networkProxiesThirdLayout">
          <item>
           <widget class="QLineEdit" name="proxyServer">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="proxyPort">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>65536</number>
            </property>
            <property name="value">
             <number>8080</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="proxyRequiresAuth">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>required password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="proxyUsername">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="proxyPassword">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab4">
       <attribute name="title">
        <string>Misc</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>File Manager</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="fileManager">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>dolphin</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>konqueror</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Browser</string>
          </property>
          <property name="buddy">
           <cstring>browser</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="browser">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Iconset</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="cbIconSet">
            <item>
             <property name="text">
              <string>default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>monochrome</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>white</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblDisconIcon">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/default/img/acid-rain-disconnected.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblIdleIcon">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/default/img/acid-rain-idle.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblBusyIcon">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/default/img/acid-rain-bussy.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblStopIcons">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/default/img/acid-rain-stopped.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="dialogButtonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../DropboxIcons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>openFileBrowser()</slot>
  <slot>setIcons()</slot>
 </slots>
</ui>
